specification/security/resource-manager/Microsoft.Security/preview/2019-01-01-preview/assessmentMetadata.json

```diff
@@ -495,16 +496,42 @@
               {
                 "value": "CustomerManaged",
                 "description": "User assessments pushed directly by the user or other third party to Azure Security Center"
+              },
+              {
+                "value": "VerifiedPartner",
+                "description": "An assessment that was created by a verified 3rd party if the user connected it to ASC"
               }
             ]
           }
+        },
+        "partnerData": {
+          "$ref": "#/definitions/SecurityAssessmentMetadataPartnerData"
```

Comment:
issues with new properties in same version..
This will break existing clients (with older SDKs), as they may not be able to parse and can potentially overwrite this field.

Reply:
same here.. breaking change guide : [https://microsoft.sharepoint.com/:w:/t/azureresourcemanagerteam/EWXsAQ1yx25KkyYCeeWGUgwBSBxEdUDEbHi6FZ__U8EOQw?e=xEcppo](https://microsoft.sharepoint.com/:w:/t/azureresourcemanagerteam/EWXsAQ1yx25KkyYCeeWGUgwBSBxEdUDEbHi6FZ__U8EOQw?e=xEcppo)

New property added to response 

If a new property/field is added to the response an API, the GET-PUT pipeline will be broken. Consider the case where from portal a customer updates the value of a new property "A". Another customer does a GET of this resource using the SDK. The SDK will ignore the property since it does not understand it. From the SDK, the customer does a PUT using the model that was returned from the GET. This will overwrite the change made by the first customer from the portal.  

---
In reply to: [367701213](https://github.com/Azure/azure-rest-api-specs/pull/8143#discussion_r367701213) [](ancestors = 367701213)

Reply:
we usually to this for new properties, but since this property cannot be added to any existing assessment and users are not, and will not be able to use PUT on the assessments that will have this property I cannot see a way for that to happen
it sounds safe to me, WDYT?

Reply:
I agree with Sanjai here that a new enum value will break existing clients because they won't be able to deserialize it. Can you add these changes in a new API version?

---
In reply to: [368309743](https://github.com/Azure/azure-rest-api-specs/pull/8143#discussion_r368309743) [](ancestors = 368309743)

Reply:
added to a new API version